kalite/distributed/static/perseus/.gitmodules

```diff
@@ -0,0 +1,15 @@
+[submodule "ke"]
```

Comment:
Do we want to have this file in here, given the subtree approach? I guess if we deleted it, it would mess up the process of pushing back to upstream (since it would be trying to delete the file for them?